inc/class-purge-cache.php

```diff
@@ -0,0 +1,183 @@
+<?php
+/**
+ * WP Irving Cache.
```

Comment:
```suggestion
 * WP Irving Cache
```
No periods in file headers, [per the core standards](https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/php/#6-file-headers).